content/docs/command-reference/repro.md

```diff
@@ -1,32 +1,37 @@
 # repro
 
-Reproduce complete or partial [pipelines](/doc/command-reference/dag) by
-executing commands defined in their [stages](/doc/command-reference/run) in the
-correct order. The commands to be executed are determined by recursively
-analyzing dependencies and <abbr>outputs</abbr> of the target stages.
+Reproduce complete or partial <abbr>pipelines</abbr> by executing commands
```

Comment:
why do we change this? is it relevant? can we do it as part of another PR?

Reply:
Just changed a link to `<abbr>`.

Reply:
But OK rolling back.

Reply:
Done.